nntrainer/src/fc_layer.cpp

```diff
@@ -139,7 +140,9 @@ Tensor FullyConnectedLayer::forwarding(Tensor in, Tensor output, int &status) {
     status = ML_ERROR_INVALID_PARAMETER;
   }
   input = in;
-  hidden = input.dot(weight).add(bias);
+  hidden = input.dot(weight);
+  hidden.add_i(bias);
+  
```

Comment:
``` hidden = input.chain().dot(weight).add(bias).run()?``` ???

Reply:
Old version has less memcopy (since current implementation of `chain()` copies in the beginning).
I changed to the suggested version and put a comment for a better readability :)

Thanks 👍